Engine/Source/ResourceMaterial.cpp

```diff
@@ -21,7 +21,7 @@ ResourceMaterial::ResourceMaterial(
     //if (mDiffuseTexture) mEnableDiffuseTexture = true;
     //else mEnableDiffuseTexture = false;
     mEnableDiffuseTexture = (mDiffuseTexture != nullptr) ? true : false;
-    mDiffuseFactor = float4(0.228f, 0.235f, 0.241, 1.0f);
+    mDiffuseFactor = float4(0.228f, 0.235f, 0.241f, 1.0f);
```

Comment:
For me those numbers looks like "magic numbers", should we add those as constants with concrete names, or this is correct as it is?